source/languages/en/riak/ops/building/installing/index.md

```diff
@@ -31,6 +31,12 @@ detailed Riak installation instructions.
 
   * [[Installing Riak With Chef]]
 
+### Cloudsoft
+
+Cloudsoft provides tested, optimized Riak blueprints to help develop and deploy applications faster and easier on a wide range of environments. The AMP-Basho software will automatically deploy and manage Basho Riak and Riak Enterprise clusters using Apache Brooklyn's easy-to-use YAML blueprinting approach, combined with Cloudsoft's enterprise-supported Application Management Platform (AMP).
```

Comment:
There's been a lot of bikeshedding around this, but I at least have been trying to maintain an 80 character column width in the docs---especially in new content added to the docs. Could you go back and break this paragraph up?